.travis.yml

```diff
@@ -5,6 +5,10 @@ bundler_args: --without development
 rvm:
   - "2.1.5"
 
+branches:
+  only:
+  - master
```

Comment:
We should add:
```yaml
  - merged-pull/*
```

Otherwise CI can't use Travis to check merged pull requests. Alternatively, we can deactivate auto-merging of pull requests and always merge/rebase manually.